IFC4x3/PropertyConstants/v/VASCO COSTA_1jQ7XYFLbCQggk_8EEnXx4.xml

```diff
@@ -1,3 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VASCO_COSTA_1jQ7XYFLbCQggk_8EEnXx4" Name="VASCO COSTA" UniqueId="6d687862-3d59-4c6a-aaae-f8838ec61ec4" />
```

Comment:
@jmirtsch  can you correct the name to VASCO_COSTA it is a required Enumeration within PEnum_AddedMassCoefficientMethod type

Reply:
This one apparently got completely deleted.

Reply:
Yes it needs reverting and only the name being corrected

Reply:
Seems to be another ghost duplicate, https://github.com/bSI-InfraRoom/IFC-Specification/tree/64f1fc0e8cdf8aba82d2b99a423ddefae2f5a2ba/IFC4x3/PropertyConstants/v

Reply:
I see it is correct.